src/pudl/metadata/resources/ferc1.py

```diff
@@ -1971,21 +2009,29 @@
         "field_namespace": "ferc1",
     },
     "out_ferc1__yearly_rate_base": {
-        "description": (
-            "This table contains granular accounting data consisting of what utilities can typically "
-            "include in their rate bases. This table is derived from seven FERC Form 1"
-            " accounting tables with nested calculations. "
-            "We reconciled these nested calculations and then identified the most "
-            "granular data across the tables.\n"
-            "We applied slight modifications to three columns (utility_type, plant_function & "
-            "plant_status) as compared to the originally reported values in our core tables. "
-            "The modifications were applied to either provide more specificity (i.e. we converted"
-            "some `total` utility_type's into `electric`) or to condense similar categories "
-            "for easier analysis (i.e. creating a `hydraulic_production` plant_function by "
-            "combining `hydraulic_production_conventional` and `hydraulic_production_pumped_storage`\n"
-            "See ``pudl.output.ferc1.Exploder`` for more details. This table was made entirely with "
-            "support and direction from RMI."
-        ),
+        "description": {
+            "additional_summary_text": "granular accounting data consisting of what utilities can typically include in their rate bases.",
+            "additional_source_text": "(Schedules 110, 118, 200, 204 219 and 320)",
+            "usage_warnings": [DETAILED_ACCOUNTING_TABLES_WARNING],
+            "additional_details_text": (
+                "This table is derived from seven FERC Form 1 "
+                "accounting tables with nested calculations. "
+                "We reconciled these nested calculations and then identified the most "
+                "granular data across the tables.\n"
+                "This table is derived three FERC Form 1 accounting tables:\n\n"
+                "* :ref:`out_ferc1__yearly_detailed_balance_sheet_assets`\n"
+                "* :ref:`out_ferc1__yearly_detailed_balance_sheet_liabilities`\n"
+                "* :ref:`core_ferc1__yearly_operating_expenses_sched320`\n\n"
```

Comment:
Is this a copypaste gremlin? I'd take it as an improvement since the original didn't list the tables, except that this bit lists three and the topic sentence mentions seven, which seems 👀 

```suggestion
```

Reply:
better? now?

Reply:
💯